specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/sensitivityLabels.json

```diff
@@ -0,0 +1,497 @@
+          "Gets the current and recommended sensitivity labels of a given database in a managed instance": {
+            "$ref": "./examples/ManagedInstanceSensitivityLabelsListByDatabaseMin.json"
+          },
+          "Gets the current sensitivity labels of a given database in a managed instance": {
+            "$ref": "./examples/ManagedInstanceSensitivityLabelsListByDatabaseCurrent.json"
+          },
+          "Gets the recommended sensitivity labels of a given database in a managed instance": {
+            "$ref": "./examples/ManagedInstanceSensitivityLabelsListByDatabaseRecommended.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SensitivityLabelProperties": {
+      "description": "Properties of a sensitivity label.",
+      "type": "object",
+      "properties": {
+        "labelName": {
+          "description": "The label name.",
+          "type": "string"
+        },
+        "informationType": {
+          "description": "The information type.",
+          "type": "string"
+        }
+      }
+    },    
+    "SensitivityLabel": {
+      "description": "A sensitivity label.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SensitivityLabelProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SensitivityLabelListResult": {
+      "description": "A list of sensitivity labels.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SensitivityLabel"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "description": "The subscription ID that identifies an Azure subscription.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "client"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The API version to use for the request.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "client"
+    },
+    "ResourceGroupParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "description": "The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ServerNameParameter": {
```

Comment:
>ServerNameParameter [](start = 5, length = 19)

this parameter everywhere was changed to `managedInstanceName`. There is nor reason to have it here anymore. I'd advice to move ManagedInstanceName definition here and then reference it everywhere in the swagger instead of defining it again and again in each operation parameter list.